packages/neo-one-server-plugin-wallet/src/bootstrap.js

```diff
@@ -0,0 +1,287 @@
+  const wallet = await getWallet(walletName, networkName, cli, plugin);
+
+  let neo;
+  let gas;
+  if (from == null) {
+    const seed = randomInt(100);
+    neo = randomIntDist(seed);
+    gas = randomIntDist(seed);
+  } else {
+    const transferPercent = randomInt(75) / 100;
+    neo = Math.ceil(transferPercent * Number(from.neoBalance));
+    gas = Math.ceil(transferPercent * Number(from.gasBalance));
+  }
+
+  return [
+    {
+      amount: new BigNumber(neo),
+      asset: common.NEO_ASSET_HASH,
+      to: wallet.address,
+    },
+    {
+      amount: new BigNumber(gas),
+      asset: common.GAS_ASSET_HASH,
+      to: wallet.address,
+    },
+  ];
+}
+
+export default (plugin: WalletPlugin) => ({ cli }: InteractiveCLIArgs) =>
+  cli.vorpal
+    .command('bootstrap', 'Bootstraps a Network with test data.')
+    .option('-n, --network <name>', 'Network to bootstrap')
+    .option('--wallets <number>', 'Number of wallets to create - default 10')
+    .action(async args => {
+      const networkName = await getNetwork({
+        cli,
+        args,
+        options: args.options,
+      });
+
+      if (
+        networkName === networkConstants.MAIN ||
+        networkName === networkConstants.TEST
+      ) {
+        throw new Error(
+          'Invalid Network: Can only bootstrap a private network',
+        );
+      }
+
+      const networkResource = await cli.client.getResource({
+        plugin: networkConstants.PLUGIN,
+        name: networkName,
+        resourceType: networkConstants.NETWORK_RESOURCE_TYPE,
+        options: {},
+      });
+      if (networkResource == null) {
+        throw new Error(`Network ${networkName} does not exist.`);
+      }
+      const network = ((networkResource: $FlowFixMe): Network);
+
+      const masterWallet = await getWallet(
+        constants.MASTER_WALLET,
+        network.name,
+        cli,
+        plugin,
+      );
+
+      if (masterWallet.wif == null) {
+        throw new Error('Something went wrong, wif is null');
+      }
+
+      const keystore = new LocalKeyStore({
+        store: new LocalMemoryStore(),
+      });
+
+      await keystore.addAccount({
+        network: network.name,
+        name: masterWallet.name,
+        privateKey: wifToPrivateKey(masterWallet.wif),
+      });
+
+      const wallets = [];
+      const numWallets = getNumWallets(args.options);
+      for (let i = 1; i < numWallets + 1; i += 1) {
+        wallets.push(`wallet-${i}`);
+      }
+
+      await Promise.all(
+        wallets.map(walletName =>
+          createWallet(walletName, network.name, cli, keystore),
+        ),
+      );
+
+      const client = new Client({
+        memory: new LocalUserAccountProvider({
+          keystore,
+          provider: new NEOONEProvider({
+            options: [
+              { network: network.name, rpcURL: network.nodes[0].rpcAddress },
+            ],
+          }),
+        }),
+      });
+      const [walletsBatch1, walletsBatch2] = _.chunk(
```

Comment:
nit: avoid appending with numbers, use full names like `firstBatch` and `secondBatch`, it's much easier to visually and mentally distinguish. Similar to not using abbreviations for variable names.